spec/features/loggingin_spec.rb

```diff
@@ -0,0 +1,37 @@
+feature "Login Page" do
+
+  context "when logging in:" do
+
+    before do
+      test_sign_up
+      click_button('Logout')
+      visit('/sessions/new')
+    end
+
+    scenario "the user should be be able to log in with their details" do
+      login
+      expect(current_path).to eq('/peeps')
+      expect(page).to have_content('Welcome to Chitter, Sinatra!')
+    end
+
+    scenario "incorrect details will fail the login process" do
+      fill_in('username', with: 'test_account')
+      fill_in('password', with: 'wrong')
+      click_button('Log in!')
+      within('div#login_failed') do
+        expect(page).to have_content('Login failed, please confirm you have entered the correct details.')
```

Comment:
Metrics/LineLength: Line is too long. [106/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)